resources/qml/PrinterOutput/MonitorButtonStyle.qml

```diff
@@ -0,0 +1,88 @@
+import QtQuick 2.10
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import QtQuick.Layouts 1.3
+
+import UM 1.2 as UM
+import Cura 1.0 as Cura
+
+
+Component
+{
+    ButtonStyle

```

Comment:
Styles should be defined in the theme. Also have a look if it's possible to ensure that we don't need a new unique style for these buttons.

Reply:
Done in 6bb010e